src/pages/resources.tsx

```diff
@@ -0,0 +1,287 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { useTranslation } from 'next-i18next';
+import {
+  chakra,
+  Input,
+  Text,
+  Button,
+  HStack,
+  VStack,
+  useToast,
+  Box,
+  Switch,
+  Wrap,
+} from '@chakra-ui/react';
+import { LinkIcon } from '@chakra-ui/icons';
+import Logo from '../components/Logo';
+import PageLayout from '../layout/Page';
+import Airtable from 'airtable';
+import KnowledgeCard from '../components/ResourceCard';
+import {
+  AIRTABLE_READONLY_KEY,
+  AIRTABLE_RESOURCE_BASE,
+} from '../utils/DeveloperDaoConstants';
+import {
+  Author,
+  Blockchain,
+  Category,
+  Resource,
+  Tag,
+} from '../utils/AirtableResourceClasses';
+
+function KnowledgeBase(props: { airtable: Airtable }) {
```

Comment:
`props` aren't actually used!

```suggestion
function KnowledgeBase() {
```